src/app/_layout.tsx

```diff
@@ -134,7 +134,7 @@ export default function Layout() {
                       : "modal"
                     : "modal",
                 sheetGrabberVisible: true,
-                sheetAllowedDetents: [0.6],
+                sheetAllowedDetents: [0.9],
```

Comment:
Otherwise we get a huge empty space at the bottom
```suggestion
                sheetAllowedDetents: [0.6, 0.9],
```